### PR TITLE
xclip.md

### LINKED ISSUE #131: xclip

### PR DESCRIPTION
**Pull request by bishengliu7**
# Advanced HW 9 submission

Closes #131 

Name: Bisheng Liu
Uniqname: bisheng

### CODE REVIEW

**Comment by bishengliu7** [comment]
when will any instructor check the pull request?

File: _commands/tools/xclip.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+#### -selection
+
+~~~ bash
+ls -la | xclip -selection clipboard
+~~~
+
+##### Break it down
+
+We can specify which X selection to use. The default is "primary". Other options are "secondary" and "clipboard". "clipboard" means the system clipboard. Thus, in the upper example, we can use Ctrl+Shift+v to paste the result later.
```
Comment (mterwill) [inline _commands/tools/xclip.md]: You can use \<kbd\>\</kbd\> to make  <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>V</kbd>

File: _commands/tools/xclip.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ls -la | xclip -selection clipboard
+~~~
+
+##### Break it down
+
+We can specify which X selection to use. The default is "primary". Other options are "secondary" and "clipboard". "clipboard" means the system clipboard. Thus, in the upper example, we can use Ctrl+Shift+v to paste the result later. 
+
+##### To make life easier
+
+As mentioned, '-selection clipboard' is more convenient than default setting, making an alias can make the life much easier:
```
Comment (mterwill) [inline _commands/tools/xclip.md]: Inline command line flags should be surrounded in \`backticks\` so they appear `like this`. Please make this change across your documentation.

**Comment by mterwill** [comment]
If there's a section on OS X's `pbcopy` merged in, link to that. Otherwise, please add a note saying if you're on a Mac the equivalent command is `pbcopy`.

**Comment by bishengliu7** [comment]
I've changed according to the requirements.

**Comment by mterwill** [comment]
Looks good

**Pull request merged**